rust/kernel/module_param.rs

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Types for module parameters.
+//!
+//! C header: [`include/linux/moduleparam.h`](../../../include/linux/moduleparam.h)
+
+use core::fmt::Write;
+
+/// Types that can be used for module parameters.
```

Comment:
```suggestion
/// Types that can be used for module parameters.
///
```

i.e. the first line/sentence (the "summary") should be separated.